### PR TITLE
Shutdown Circuit

### PR DESCRIPTION
**Pull request by rileychapman**

### CODE REVIEW

File: esf_main.tex [hunk 1]
```diff
@@ -134,13 +134,30 @@ \section{System Overview}
...
 
 \subsection{Shutdown Circuit}
 
 \subsubsection{Description/concept}
 %Describe your concept of the shutdown circuit, the master switches, shut down buttons, brake over travel switch, etc.
 
+The shutdown circuit directly controls the current that closes the AIRs. Since each switch/relay on the shutdown circuit is connected in series, that current must pass through every switch and safety check. If any device on the circuit is triggered by an unsafe condition, the current to the AIRs is cut off and the AIRs open, disconnecting the accumulator from the tractive system. Any capacitive load on the tractive system outside the accumulator is then discharged to ensure that all high voltage is contained inside the accumulator. 
+
+	\begin{itemize}
+		\item The GLV Master Switch (GLVMS) controls power to the entire low voltage system. As a result, high voltage cannot be present when the low voltage system is not active. The GLVMS has a lockout such that the vehicle can be safely worked on.
```
Comment (aehoppe) [inline esf_main.tex]: Including the AIRs

File: esf_main.tex [hunk 2]
```diff
@@ -134,13 +134,30 @@ \section{System Overview}
...
 \subsection{Shutdown Circuit}
 
 \subsubsection{Description/concept}
 %Describe your concept of the shutdown circuit, the master switches, shut down buttons, brake over travel switch, etc.
 
+The shutdown circuit directly controls the current that closes the AIRs. Since each switch/relay on the shutdown circuit is connected in series, that current must pass through every switch and safety check. If any device on the circuit is triggered by an unsafe condition, the current to the AIRs is cut off and the AIRs open, disconnecting the accumulator from the tractive system. Any capacitive load on the tractive system outside the accumulator is then discharged to ensure that all high voltage is contained inside the accumulator. 
+
+	\begin{itemize}
+		\item The GLV Master Switch (GLVMS) controls power to the entire low voltage system. As a result, high voltage cannot be present when the low voltage system is not active. The GLVMS has a lockout such that the vehicle can be safely worked on. 
+		\item The Left E-Stop is a large red push-rotate button located at approximately the hight of the driver's head on the left side of the vehicle.
```
Comment (aehoppe) [inline esf_main.tex]: *height

File: esf_main.tex [hunk 3]
```diff
@@ -134,13 +134,30 @@ \section{System Overview}
...
+
+	\begin{itemize}
+		\item The GLV Master Switch (GLVMS) controls power to the entire low voltage system. As a result, high voltage cannot be present when the low voltage system is not active. The GLVMS has a lockout such that the vehicle can be safely worked on. 
+		\item The Left E-Stop is a large red push-rotate button located at approximately the hight of the driver's head on the left side of the vehicle. 
+		\item The Right E-Stop is identical to the Left E-Stop located on the other side of the vehicle. 
+		\item The Brake System Plausibility Device (BSPD) holds a normally open relay closed when it has not detected an implausibility. When it does detect an implausibility, it latches in that state until the GLV system is power cycled. This shuts down the Tractive System in the event that the brake is pressed while the motors are being significantly driven.
```
Comment (aehoppe) [inline esf_main.tex]: it latches in a low state

File: esf_main.tex [hunk 4]
```diff
@@ -134,13 +134,30 @@ \section{System Overview}
...
+		\item The Brake Over Travel Switch (BOTS) is a push pull switch located behind the brake pedal. If the pedal over travels, the switch is pushed into an open state and stays that way until it is pulled back into the closed state. This shuts down the Tractive System in the event that brake pressure is lost. 
+		\item The Dashboard E-stop functions the same as the other E-stops. It is located such that it can be easily pressed by the driver. 
+		\item The Inertia Switch operates as a crash sensor that opens the shutdown circuit in the event that it detects an acceleration indicative of a collision. This ensures that the vehicle is electrically safe in an emergency situation. 
+		\item The Battery Management System (BMS) monitors the condition of the accumulator. It holds a normally open relay closed while the Lithium cells are operating within safe temperature and electrical conditions.
```
Comment (aehoppe) [inline esf_main.tex]: Lithium doesn't need to be capitalized.

File: esf_main.tex [hunk 5]
```diff
@@ -161,18 +178,23 @@ \subsubsection{Description/concept}
     \end{table}
 
 \subsubsection{Wiring / additional circuitry}
-%shutdown schematic
-TODO
+	Figure \ref{shutdown_circuit} shows the wiring of the shutdown circuit.
+
+	\begin{sidewaysfigure}[p]
+        \includegraphics[width= 1\textheight]{shutdown_circuit}
+        \caption{Block Diagram of Shutdown Circuit}
```
Comment (aehoppe) [inline esf_main.tex:201]: We're going to have to figure out how to make a numbered list of figures somewhere down the line. I'll look at last year's ESF for reference

**Review by aehoppe — CHANGES_REQUESTED** [review]
Looks good. I left a few comments on small errors or things that could be a little clearer.

File: esf_main.tex (hunk 5 above)
Comment (rileychapman) [reply esf_main.tex:201]: The list of figures should be generated automatically

**Comment by rileychapman** [comment]
Made edits based on comments, also the work I have done on motor and motor controller have made it into this pull request

**Comment by aehoppe** [comment]
Looks good to me

**Pull request merged**